tox.ini

```diff
@@ -13,5 +13,5 @@ whitelist_externals =
 commands=
     python ./scripts/license_verify.py
     python setup.py check -r -s
-    pylint azdev --rcfile=.pylintrc -r n
+    pylint azdev --rcfile=.pylintrc -r n --load-plugins=pylint.extensions.no_self_use
```

Comment:
Add `load-plugins` to prevent `azdev/operations/help/refdoc/common/directives.py:46:0: R0022: Useless option value for 'disable', 'no-self-use' was moved to an optional extension, see https://pylint.pycqa.org/en/latest/whatsnew/2/2.14/summary.html#removed-checkers. (useless-option-value)`

Reply:
Will it make more sense if we remove `# pylint: disable=no-self-use` instead? Pylint apparently doesn't think `no-self-use` is important anymore: https://github.com/pylint-dev/pylint/issues/5502

Reply:
Yes, we do this in #359